Documentation/rollup-reports.md

```diff
@@ -31,7 +31,7 @@ It contains a few a custom inputs: most importantly, `NamespaceCPUUsageReportNam
 apiVersion: metering.openshift.io/v1
 kind: ReportQuery
 metadata:
-  name: namespace-cpu-usage
+  name: custom-namespace-cpu-usage
```

Comment:
@timflannagan1 if this doesn't exists, I'm afraid it will cause confusion for the end-users.

Reply:
You would be creating this query, not referencing an existing one.